### ISSUE TITLE
CONTROL+ALT+N shortcut much slower at starting NVDA than other methods

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#16682]
### Steps to reproduce:

1. Time how long NVDA takes to start when pressing CONTROL+ALT+N
2. Exit NVDA
3. Time how long NVDA takes to start when activating a desktop or taskbar shortcut, or from the WINDOWS+R dialog.

### Actual behavior:

NVDA takes ~5-7 seconds to start when using control+alt+n (timed with a stopwatch).
NVDA takes ~2-3 seconds to start when using any other method.

### Expected behavior:

Using the shortcut should be as quick as any other way of starting NVDA.

#### NVDA version:

NVDA 2024.2 Beta 3 Installed

#### Windows version:

Windows 11 (64-bit) Version: 23H2, Build: 22631.3593 

#### Other information about your system:

I tested exactly the same test with starting Microsoft Word via a desktop shortcut set to use control+alt+w and had basically identical results to NVDA.  The issue is a Windows bug, not anything to do with NVDA.  I'm only writing it up to track any update from Microsoft and for anyone else reporting this issue to us to find.

(Reported to Microsoft on 12 June 2024)
